src/Helpers/OAuth2/OAuthConfig.php

```diff
@@ -224,6 +232,38 @@ public function setDefaultScopes(array $defaultScopes): static
         return $this;
     }
 
+    /**
+     * Set the request modifier callable which can be used to modify the request being sent
+     *
+     * @param callable(\Saloon\Contracts\Request): (void) $requestModifier
+     * @return $this
+     */
```

Comment:
We use this in the relevant traits, as well.
As long as the template is used in a parameter, in one way or another, it should be fine.
We'll see if PHPStan thinks otherwise. 👍 
Doing this change, did solve complaints it had for me in earlier PRs, though.

```suggestion
    /**
     * Set the request modifier callable which can be used to modify the request being sent
     *
     * @template TRequest of \Saloon\Contracts\Request
     *
     * @param callable(TRequest): (void) $requestModifier
     * @return $this
     */
```